src/cryptographic_primitives/proofs/sigma_correct_homomorphic_elgamal_enc.rs

```diff
@@ -8,13 +8,12 @@
 
 use serde::{Deserialize, Serialize};
 
-use digest::Digest;
-use sha2::Sha256;
-
 use crate::cryptographic_primitives::hashing::DigestExt;
 use crate::elliptic::curves::{Curve, Point, Scalar};
+use digest::Digest;
 
 use super::ProofError;
+use std::marker::PhantomData;
```

Comment:
It's only my preference - see at use statements order. I usually use this order: `std -> external libs -> crate::* imports -> super::* import`. Sometimes `std -> macro imports -> external libs -> ...`. I wish it could be embedded to `cargo fmt`

Reply:
I use CLion -> Code -> Optimize Imports :) (requires to first remove the new lines between all the imports and then it adds them back where it see fits)
But I can reorder

Reply:
yep, too bad rustfmt doesn't do something like that